src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -526,7 +527,10 @@ class Shuttle (
             val flightNames = flightPlan.keys.joinToString(", ") { it.name }
             val exceptionLog = "Encountered exception $ex while integrating flight plan containing flight(s) $flightNames"
             writeLog(flightNames, setOf(exceptionLog, ex.stackTrace.toString()), IntegrationStatus.FAILED)
-        } finally {
+            if (!isShuttleServer) {
+                MissionControl.fail(1, flightPlan.keys.first(), ex, listOf(uploadingExecutor))
```

Comment:
also bump on this comment https://github.com/openlattice/shuttle/pull/146#discussion_r381505927